azure-toolkit-libs/azure-toolkit-appservice-lib/src/main/java/com/microsoft/azure/toolkit/lib/appservice/service/impl/AppServiceUtils.java

```diff
@@ -81,8 +81,12 @@ private static Runtime getRuntimeFromLinuxFunctionApp(String linuxFxVersion) {
 
     private static Runtime getRuntimeFromWindowsAppService(WebAppBase webAppBase) {
         final String javaContainer = String.join(" ", webAppBase.javaContainer(), webAppBase.javaContainerVersion());
-        final WebContainer webContainer = WebContainer.fromString(javaContainer);
-        final JavaVersion javaVersion = JavaVersion.fromString(Objects.toString(webAppBase.javaVersion(), null));
+        final WebContainer webContainer = StringUtils.isAllEmpty(webAppBase.javaContainer(), webAppBase.javaContainerVersion()) ?
```

Comment:
StringUtils.isEmpty(webAppBase.javaContainer()) -> WebContainer.Java_Off

if (webContaikner != WebContainer.Java_Off) -> parseJavaVersion from webAppBase.javaVersion()

JavaVersion shall handle 'OFF' value
then remove this line .filter(version -> !Objects.equals(version, com.azure.resourcemanager.appservice.models.JavaVersion.OFF))